domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/EvidenceType.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+public enum EvidenceType {
```

Comment:
we have 3 classes with this definition. Why don't we add it to one submodule and reuse?

eg somewhere here: uk.gov.hmcts.cmc.domain.[evidence] 

I have added "[evidence]" as it doesn't exist yet, but we have submodule domain-model submodule where we should store this kind of data IMO.

Reply:
You mean `sub-package`?